Control/test/lib/mocha-status-service.js

```diff
@@ -156,14 +156,14 @@ describe('StatusService test suite', () => {
 
     it('should successfully retrieve status and info about Kafka that it is not running', async () => {
       const status = new StatusService(config, {}, {});
-      const kafkaStatus = await status.getKafkaStatus(new KafkaConnector(config.kafka));
+      const kafkaStatus = await status.getNotificationStatus(new NotificationService(config.kafka));
       expectedInfo.status = {ok: false, configured: true, message: 'KafkaJSNumberOfRetriesExceeded'};
       assert.deepStrictEqual(kafkaStatus, expectedInfo);
     }).timeout(5000);
 
     it('should successfully return that Kafka was not configured if configuration is not provided', async () => {
       const status = new StatusService({}, {}, {});
-      const kafkaStatus = await status.getKafkaStatus(new KafkaConnector());
+      const kafkaStatus = await status.getNotificationStatus(new NotificationService());
```

Comment:
Could you please make sure all `kafka` naming occurrences are updated to avoid future confusion?
```suggestion
      const notificationStatus = await status.getNotificationStatus(new NotificationService());
```